.babelrc

```diff
@@ -12,6 +12,7 @@
       }
     ],
     "react",
-    "stage-2"
+    "stage-2",
+    "stage-0"
```

Comment:
What stage-0 features are we using?

Reply:
I thought it was required for async/await, maybe its just the babel polyfill I needed

Reply:
removed it, wasn't needed.